lib/vagrant-service-manager/binary_handlers/adb_binary_handler.rb

```diff
@@ -4,52 +4,6 @@ class ADBBinaryHandler < BinaryHandler
       def initialize(machine, env, options)
         super(machine, env, options)
       end
-
```

Comment:
I am a bit surprised that you pull that in binary handler, especially since you rightfully extend the CDK handlers from the ADB ones.